src/main/resources/entity-types/inventory/simple_holdings_records.json5

```diff
@@ -836,6 +836,25 @@
       visibleByDefault: false,
       valueGetter: ":sourceAlias.jsonb->>'notes'",
     },
+    {
+      name: 'tenant_id',
+      sourceAlias: 'inst',
```

Comment:
It does feel a little weird to have 
sourceAlias: 'inst' 
here, but since there isn't really a source alias for this field, and since I think things break if we don't at least have something defined for the field, it's probably ok.

Reply:
Oh yeah :D Good catch!